<launch>

  <!-- Arguments -->
  <arg name="dataset_path" default="/home/bjoshi/shipwreck/mav0" />
  <arg name="bag" default="/home/bjoshi/shipwreck_stavronikita_sync.bag" />
  <arg name="left_image_topic" default="/cam_fl/image_raw/compressed" />
  <arg name="right_image_topic" default="/cam_fr/image_raw/compressed" />
  <arg name="imu_topic" default="/imu/imu" />

  <node name="bag_to_asl" pkg="utils_ros" type="bag_to_asl.py" output="screen">

    <param name="dataset_path" value="$(arg dataset_path)" />
    <param name="bag" value="$(arg bag)" />
    <param name="left_image_topic" value="$(arg left_image_topic)" />
    <param name="right_image_topic" value="$(arg right_image_topic)" />
    <param name="imu_topic" value="$(arg imu_topic)" />

  </node>

</launch>